txp/Pos/Txp.hs

```diff
@@ -0,0 +1,7 @@
+-- | Txp system reexports.
+
+{-# OPTIONS_GHC -F -pgmF autoexporter #-}
+
+import           Pos.Arbitrary.Txp         ()
+import           Pos.Arbitrary.Txp.Network ()
```

Comment:
`Pos.Arbitrary.Txp.Network` imports `Pos.Arbitrary.Txp`, so the import of `Pos.Arbitrary.Txp` is likely redundant. On the other hand, it's kinda weird that this dependency is not the other way around 😕 
Anyway, it's not important. I'd like `Pos.Arbitrary.Txp` to import `Pos.Arbitrary.Txp.Network`, but not too much.

Reply:
> I'd like Pos.Arbitrary.Txp to import Pos.Arbitrary.Txp.Network

me too, but I like the green checks so I might fix it in a different PR